File: src/main/java/guru/xgm/angelscript/psi/ZincTokenSets.java
package guru.xgm.angelscript.psi;

import com.intellij.psi.tree.TokenSet;

import static guru.xgm.angelscript.psi.ZincTypes.*;

public interface ZincTokenSets {
    TokenSet COMMENTS = TokenSet.create(SINGLE_LINE_COMMENT, MULTI_LINE_COMMENT);
}

File: src/main/kotlin/raft/war/language/zinc/psi/ZincTokenSets.kt
package raft.war.language.zinc.psi

import com.intellij.psi.tree.TokenSet
import raft.war.language.zinc.psi.ZincTypes.BLOCK_COMMENT
import raft.war.language.zinc.psi.ZincTypes.LINE_COMMENT

interface ZincTokenSets {
    companion object {
        val COMMENTS: TokenSet = TokenSet.create(LINE_COMMENT, BLOCK_COMMENT)
    }
}
